src/App.js

```diff
@@ -98,6 +99,10 @@ class App extends Component {
                             path="/terms"
                             render={props => <Terms {...props} me={me} />}
                         />
+                        <Route
```

Comment:
What is required to run the admin API and views as a separate application?
This is not the best practice for admins.